Source/Scene/GltfLoaderUtil.js

```diff
@@ -104,6 +106,21 @@ GltfLoaderUtil.createSampler = function (options) {
     usesTextureTransform = true;
   }
 
+  if (
+    compressedTextureNoMipmap &&
+    minFilter !== TextureMinificationFilter.LINEAR &&
+    minFilter !== TextureMinificationFilter.NEAREST
+  ) {
+    if (
+      minFilter === TextureMinificationFilter.NEAREST_MIPMAP_NEAREST ||
+      minFilter === TextureMinificationFilter.NEAREST_MIPMAP_LINEAR
+    ) {
+      minFilter = TextureMinificationFilter.NEAREST;
+    } else {
+      minFilter = TextureMinificationFilter.LINEAR;
+    }
+  }
+
   if (
```

Comment:
Should the `usesTextureTransform` be an else-if here? Not sure how the extension would interact with KHR_texture_basisu.

Reply:
They are orthogonal to each other so what you have is fine. Though it could all be one if statement:

```js
  if (
    (compressedTextureNoMipmap || usesTextureTransform) &&
    minFilter !== TextureMinificationFilter.LINEAR &&
    minFilter !== TextureMinificationFilter.NEAREST
  ) {
    if (
      minFilter === TextureMinificationFilter.NEAREST_MIPMAP_NEAREST ||
      minFilter === TextureMinificationFilter.NEAREST_MIPMAP_LINEAR
    ) {
      minFilter = TextureMinificationFilter.NEAREST;
    } else {
      minFilter = TextureMinificationFilter.LINEAR;
    }
  }
```